models/Books.js

```diff
@@ -0,0 +1,12 @@
+
+const Books = (connection, Sequelize) => {
```

Comment:
You are missing your Authors model association
const Books = (connection, Sequelize, Authors) => {

authorId: { type: Sequelize.INTEGER, references: { model: Authors, key: 'id' } },